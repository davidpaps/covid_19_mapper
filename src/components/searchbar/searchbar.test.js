import * as React from "react";
import { shallow } from "enzyme";
import SearchContainer from "./searchbar";
import CountryGraphContainer from "../graphs/country_graphs/country_graph";

describe("Searchbar", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<SearchContainer />)));

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(3);
  });

  it("should render a <form />", () => {
    expect(wrapper.find("form").length).toEqual(1);
  });

  it("should render the CountryGraphContainer Component", () => {
    expect(wrapper.containsMatchingElement(<CountryGraphContainer />)).toEqual(
      true
    );
  });

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
