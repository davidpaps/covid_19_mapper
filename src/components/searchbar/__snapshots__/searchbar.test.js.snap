// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Searchbar renders correctly 1`] = `ShallowWrapper {}`;
